app/helpers/application_helper/toolbar/container_group_center.rb

```diff
@@ -58,6 +58,17 @@ class ApplicationHelper::Toolbar::ContainerGroupCenter < ApplicationHelper::Tool
           'pficon pficon-edit fa-lg',
           N_('Edit Tags for this Pod'),
           N_('Edit Tags')),
+        button(
+          :container_group_protect,
+          'pficon pficon-edit fa-lg',
+          N_('Manage Policies for this #{ui_lookup(:table=>"container_group")}'),
```

Comment:
this will not work any more.

You need a proc if you want to do interpolations. And the preferred way it to avoid the `ui_lookup`. Just put "Container Group" or whatever it is called in.